packages/common/lib/common/tools.js

```diff
@@ -3095,6 +3146,11 @@ async function setExecutableCapabilities(execPath, capabilities, modeEffective,
     }
 }
 
+function showDeprecatedMessage(func, logger) {
```

Comment:
as mentioned above I would assume we do not need it

Reply:
Will be deleted later. When all CLI command  will be converted.